src/BlockEntities/FurnaceEntity.h

```diff
@@ -84,6 +84,9 @@ class cFurnaceEntity :
 	/** Returns true if there's time left before the current fuel is depleted */
 	bool HasFuelTimeLeft(void) const { return (GetFuelBurnTimeLeft() > 0); }
 
+	/** Calculates and returns the experience reward for the current recipe */
+	int GetReward(void);
```

Comment:
This adds the function into the plugin interface (it is in the `tolua` block). You will need to add the function documentation to the API description files in ~/Server/Plugins/APIDump/Classes/*.lua. See the ~/Server/Plugins/APIDump/APIDesc.lua file for details on the format that is understood by the system.